gomint-server/src/main/java/io/gomint/server/command/vanilla/GamemodeCommand.java

```diff
@@ -0,0 +1,93 @@
+package io.gomint.server.command.vanilla;
+
+import io.gomint.command.Command;
+import io.gomint.command.CommandOutput;
+import io.gomint.command.CommandSender;
+import io.gomint.command.ConsoleCommandSender;
+import io.gomint.command.annotation.Description;
+import io.gomint.command.annotation.Name;
+import io.gomint.command.annotation.Parameter;
+import io.gomint.command.annotation.Permission;
+import io.gomint.command.annotation.Overload;
+import io.gomint.command.validator.TargetValidator;
+import io.gomint.command.validator.EnumValidator;
+import io.gomint.command.validator.IntegerValidator;
+import io.gomint.server.entity.EntityPlayer;
+import io.gomint.world.Gamemode;
+
+import java.util.Map;
+
+/**
+ * @author lukeeey
+ * @version 1.0
+ */
+@Name( "gamemode" )
+@Description( "Sets a player's game mode." )
+@Permission( "gomint.command.gamemode" )
+@Overload( {
+    @Parameter( name = "mode", validator = EnumValidator.class, arguments = { "a", "adventure", "c", "creative", "s", "survival", "sp", "spectator" } ),
+    @Parameter( name = "player", validator = TargetValidator.class, optional = true )
+} )
+@Overload( {
+    @Parameter( name = "mode", validator = IntegerValidator.class ),
+    @Parameter( name = "player", validator = TargetValidator.class, optional = true )
+} )
+public class GamemodeCommand extends Command {
+
+    @Override
+    public CommandOutput execute( CommandSender sender, String alias, Map<String, Object> arguments ) {
+        CommandOutput output = new CommandOutput();
+        EntityPlayer target;
+
+        if( sender instanceof ConsoleCommandSender ){
+            if( arguments.containsKey( "player" ) ) {
+                target = (EntityPlayer) arguments.get( "player" );
+            } else {
+                return output.fail( "No targets matched selector" );
+            }
+        } else {
+            if ( arguments.containsKey( "player" ) ) {
+                target = (EntityPlayer) arguments.get( "player" );
+            } else {
+                target = (EntityPlayer) sender;
+            }
+        }
+
+        Gamemode mode;
+        switch( String.valueOf( arguments.get( "mode" ) ) ) {
+			case "0":
```

Comment:
indention

Reply:
Im not worried about that right now because it wont even load, and that indentation is correct in InteliJ

Reply:
The problem is that spaces and tabs are mixed in this file. Because github doesn#T display tabs as 4 spaces such code style issues are visible.